lib/purl_fetcher/client/reader.rb

```diff
@@ -18,11 +19,17 @@ def collection_members(druid)
     end
   end
 
+  # @return [Array<Hash<String,String>>] a list of hashes where the key is a digest and the value is a filepath/filename
+  def files_by_digest(druid)
+    get("/purls/druid:#{druid.delete_prefix('druid:')}", {})
```

Comment:
I didn't include the `to_enum` jazz here (like in `#collection_members` above), but it seems harmless enough if anyone wants it added.